compiler/rustc_session/src/lint/builtin.rs

```diff
@@ -1891,6 +1891,17 @@ declare_lint! {
     "detects invalid HTML tags in doc comments"
 }
 
+declare_lint! {
+    /// The `automatic_links` lint detects when a URL/email address could be
```

Comment:
```suggestion
    /// The `automatic_links` lint detects when a URL/email address could be
```
Is there any reason you included email addresses in the lint? That seems kind of niche.

Reply:
Arf, I was too much into the markdown spec and they said that the angle brackets were for both so I just went with the flow.